### PR TITLE
Home text3/cdw42

### PR DESCRIPTION
**Pull request by mrgiles1**
After struggling with conflicts for some time, decided to revert to previous commit, rebase, then force push. Hopefully all good.

### CODE REVIEW

File: frontend/src/components/FeatureSection.tsx [hunk 1]
```diff
@@ -1,53 +1,32 @@
...
 interface FeatureSectionProps {
-    headline:string
-    content:string
-    buttonLabel:string
-    buttonUrl:string
+    FEATURE_HEADLINE:string
+    FEATURE_CONTENT:string
+    FEATURE_LINK_LABEL:string
+    FEATURE_URL:string
 }
```
Comment (jtabba) [inline frontend/src/components/FeatureSection.tsx:10]: These stay camelCase, only unchanging variables use uppercase

File: frontend/src/pages/home/constants.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    },
+    {
+        HOW_IT_WORKS_HEADING: "Community and Support",
+        HOW_IT_WORKS_CONTENT: "MentorShip is more than a platform—it's a community of passionate developers, united by a love for coding and learning. Forge lasting connections, share insights, and contribute to the growth of fellow developers.",
+        HOW_IT_WORKS_LINK_LABEL: "Learn more... >",        
+        HOW_IT_WORKS_URL: "testing3"
+        
+    }
+];
+
```
Comment (jtabba) [inline frontend/src/pages/home/constants.ts:75]: Small misunderstanding here Danny, only variable names like `subFeaturesProps` or `missionSectionProps` should be using uppercase. Everything else (including interfaces) stays as camelCase

**Review by jtabba — CHANGES_REQUESTED** [review]

File: frontend/src/components/FeatureSection.tsx (hunk 1 above)
Comment (mrgiles1) [reply frontend/src/components/FeatureSection.tsx:10]: fixed

File: frontend/src/pages/home/constants.ts (hunk 2 above)
Comment (mrgiles1) [reply frontend/src/pages/home/constants.ts:75]: ok, fixed

**Review by jtabba — APPROVED** [review]

**Pull request merged**